openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -82,11 +82,19 @@ def get(self, request, username):
         """
         GET /api/user/v0/accounts/{username}/
         """
-        existing_user, existing_user_profile = self._get_user_and_profile(username)
+        account_settings = AccountView.get_serialized_account(username)
+        return Response(account_settings)
+
+    @staticmethod
+    def get_serialized_account(username):
+        """
+        Returns the serialized JSON for the specified username.
```

Comment:
Add in the documentation that callers must do authentication/permissions check before calling this method.